Sming/Components/crypto/component.mk

```diff
@@ -0,0 +1,3 @@
+COMPONENT_LIBNAME :=
+COMPONENT_DEPENDS	:= axtls-8266
```

Comment:
This dependency should probably depend on SMING_ARCH (unnecessary for ESP8266, as it uses ROM functions)

Reply:
We need axtls for other hashes... plus bearssl now...